doc/Makefile

```diff
@@ -7,7 +7,7 @@
 # Use explicit "version_info" indexing since make cannot handle colon characters, and
 # evaluate it now to allow easier debugging when printing the varaible
 
-PYVER:=$(shell python3 -c 'from sys import version_info as v; print("{0}.{1}".format(v[0], v[1]))')
+PYVER:=$(shell python3.7 -c 'from sys import version_info as v; print("{0}.{1}".format(v[0], v[1]))')
```

Comment:
this won't work for everyone, but neither did the old code (clearly not for you). so let's just merge this, and fix it properly at some point by the thing numpy did here (I _think_ that works now in master, but it was a major pain to get right)